commit 5f1314d188c4ecc2e0d5be54bcd663013e8b0077
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Tue Sep 22 13:00:34 2009 +0000

    Fix libva-x11 dependencies. Ensure this grabs libva so that people don't have to check for that themselves.

diff --git a/libva-x11.pc.in b/libva-x11.pc.in
index 75e4f0b..6dde07f 100644
--- a/libva-x11.pc.in
+++ b/libva-x11.pc.in
@@ -6,6 +6,7 @@ display=x11
 
 Name: libva-${display}
 Description: Userspace Video Acceleration (VA) ${display} interface
+Requires: libva
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -lva-${display}
 Cflags: -I${includedir}
